shadowsocks/crypto/aead.py

```diff
@@ -153,8 +153,8 @@ def __init__(self, cipher_name, key, iv, op):
         # n, n > 0, waiting data
         self._chunk = {'mlen': AEAD_MSG_LEN_UNKNOWN, 'data': b''}
 
-        self.encrypt_once = self.aead_encrypt
```

Comment:
This 2 lines should not be commented 'coz it is required for the program.
E.g. `shadowsocks/crypto/util.py` Line 126